docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,294 @@
+# Mobile exercises
+
+Writing a DataCamp for Mobile is quite different than writing a DataCamp
+[desktop course](../../courses/readme.md). The upside is that you can reach a
+much broader audience because the convenience is higher and the barrier to entry
+is lower. The challenge is that there are many more constraints to work
+within. Primary among these are the restricted screen real estate and the lack
+of an interactive interpreter. These constraints require you to think carefully
+about the way you introduce concepts and the examples that you use.
+
+# Basic exercise structure
+
+There are roughly 5 exercise types available to you as an author on DataCamp
```

Comment:
I think you need to discuss the different exercise types somewhere near here, before you get into the yaml structure.

Reply:
Moved the list of types up to the top.